### PR TITLE
Merging 1.0 changes into main

### PR DESCRIPTION
**Pull request by Kelvinrr**

### CODE REVIEW

**Comment by Kelvinrr** [comment]
2 tests failing, coming back to this tonight.

File: recipe/meta.yaml
[the remark this answers is not in the document]
Comment (Kelvinrr) [reply recipe/meta.yaml:1]: We really should after that PR is merged.

File: .github/workflows/ci_testing.yml [hunk 1]
```diff
@@ -8,17 +8,17 @@ env:
 
 jobs:
   build-library:
-
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
```
[the remark this answers is not in the document]
Comment (Kelvinrr) [reply .github/workflows/ci_testing.yml:16]: I should put that back in, that was part of me testing out why python was failing in a weird way.

[the remark this answers is not in the document]
Comment (Kelvinrr) [reply .github/workflows/ci_testing.yml]: ugh, it fails on 3.8. I think in the spirit of just getting this merged so I can move on to other things Im going to disable and make an issue of it.

**Review by chkim-usgs — APPROVED** [review]

**Pull request merged**